dr_ensenso/CMakeLists.txt

```diff
@@ -15,6 +15,13 @@ find_package(Eigen REQUIRED)
 find_package(OpenCV REQUIRED)
 find_package(PCL REQUIRED)
 
+find_package(OpenMP REQUIRED)
+if(OPENMP_FOUND)
+	message(STATUS "OPENMP FOUND")
+	set(OpenMP_FLAGS ${OpenMP_CXX_FLAGS})  # or if you use C: ${OpenMP_C_FLAGS}
+	set(OpenMP_LIBS gomp)
```

Comment:
What are these two lines for? Why not use `${OpenMP_CXX_FLAGS}` directly below? And is the explicit linking against `gomp` needed? And shouldn't we make sure the library exists before blindly linking against it?